test/conformance/test_suite_test.go

```diff
@@ -42,7 +43,7 @@ func TestTest(t *testing.T) {
 
 	rr := []Reporter{}
 	if junitPath != nil {
-		rr = append(rr, reporters.NewJUnitReporter(*junitPath))
+		rr = append(rr, ptpReporter.NewPTPJUnitReporter(*junitPath))
```

Comment:
Can you add a TODO comment about the change here when this is upgraded to ginkov2?

Reply:
Done.